src/components/IncrementStepper/IncrementStepper.jsx

```diff
@@ -0,0 +1,72 @@
+import { React, useCallback, useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { HiPlusSm, HiMinusSm } from "react-icons/hi";
+
+export const IncrementStepper = ({
+  min,
+  max,
+  initialValue,
+  label,
+  onClick,
+  ...props
+}) => {
+  const [value, setValue] = useState(initialValue);
+  console.log("rerendering component");
+  console.log("current value is: ", value);
+
+  const onIncrease = useCallback(() => {
```

Comment:
Macht das Sinn in so einer simplen Component useCallback zu benutzen? Weil die Componente an sich muss ja eh rerendert werden, weil die Zahl sich ja ändert, die sie ja gleichzeitig darstellt. Denke kaum das man hier etwas an Perfomance steigern kann. 

Oder verstehe ich useCallback falsch?

Reply:
Ja, hier ist es vermutlich wirklich quatsch. Denn so oft klicken die die buttons vermutlich nicht.